src/containers/CandidateListPage/TestList/index.js

```diff
@@ -35,7 +35,7 @@ class TestList extends React.Component {
 
   handleOnOkDelConfirmModal = async () => {
     const { delTest } = this.state;
-    const { deleteTestAction } = this.props;
+    const { deleteTestAction } = this.props.deleteTestAction;
```

Comment:
this looks weird. I don't think there's another `deleteTestAction` in `this.props.deleteTestAction`